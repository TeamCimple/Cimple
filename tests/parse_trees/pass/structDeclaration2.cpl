struct Employee {
	int age;
	int salary;
};


int main(int argc){
	struct Employee prof;
	return 0;
}
