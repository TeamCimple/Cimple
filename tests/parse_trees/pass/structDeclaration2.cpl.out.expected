Struct((int Var(age),), (int Var(salary),), , Employee, , )FuncDecl(Name: Var(main) ReturnType: int Parameters: PARAM(int Var(argc))  Body: CompoundStatement(Declarations: (Employee Var(prof),),  StatementList: RETURN(Int(0)))) 
